examples/step-7/doc/results.dox

```diff
@@ -187,7 +187,7 @@ here) that is written into a file in a way so that it could be
 copy-pasted into a LaTeX document.
 
 
-<h4> When is the error "small"? </h4>
+<h3> When is the error "small"? </h4>
```

Comment:
```suggestion
<h3> When is the error "small"? </h3>
```

Reply:
Ugh, you'd think one can't make a mistake in this short a patch. But you'd be wrong...

Fixed now.